lib/tasks/data_migration.rake

```diff
@@ -170,4 +170,80 @@ namespace :data_migration do
 
     puts "✅  Balance component migration complete."
   end
+
+  desc "Migrate global provider settings to family-specific"
```

Comment:
Lunch Flow was never in a `stable` release so we should probably open an issue to remove this `rake` task shortly after we release, right?